src/pages/settings/Profile/PersonalDetails/DateOfBirthPage.js

```diff
@@ -0,0 +1,118 @@
+import _ from 'underscore';
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import HeaderWithCloseButton from '../../../../components/HeaderWithCloseButton';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import ROUTES from '../../../../ROUTES';
+import Form from '../../../../components/Form';
+import ONYXKEYS from '../../../../ONYXKEYS';
+import * as ValidationUtils from '../../../../libs/ValidationUtils';
+import styles from '../../../../styles/styles';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import * as PersonalDetails from '../../../../libs/actions/PersonalDetails';
+import compose from '../../../../libs/compose';
+import DatePicker from '../../../../components/DatePicker';
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** User's private personal details */
+    privatePersonalDetails: PropTypes.shape({
+        dob: PropTypes.string,
+    }),
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    privatePersonalDetails: {
+        dob: '',
+    },
+};
+
+class DateOfBirthPage extends Component {
+    constructor(props) {
+        super(props);
+
+        this.validate = this.validate.bind(this);
+        this.updateDateOfBirth = this.updateDateOfBirth.bind(this);
+    }
+
+    /**
+     * Submit form to update user's first and last legal name
+     * @param {Object} values
+     * @param {String} values.dob - date of birth
+     */
+    updateDateOfBirth(values) {
+        PersonalDetails.updateDateOfBirth(
+            values.dob.trim(),
+        );
+    }
+
+    /**
+     * @param {Object} values
+     * @param {String} values.dob - date of birth
+     * @returns {Object} - An object containing the errors for each inputID
+     */
+    validate(values) {
+        const errors = {};
+        const minimumAge = 5;
+        const maximumAge = 150;
+
+        if (_.isEmpty(values.dob)) {
+            errors.dob = this.props.translate('common.error.fieldRequired');
+        }
+        const dateError = ValidationUtils.getAgeRequirementError(values.dob, minimumAge, maximumAge);
+        if (dateError) {
+            errors.dob = dateError;
+        }
+
+        return errors;
+    }
+
+    render() {
+        const privateDetails = this.props.privatePersonalDetails || {};
+
+        return (
+            <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+                <HeaderWithCloseButton
+                    shouldShowBackButton
```

Comment:
Missing `title` here

Reply:
yeahhhh this is actually the design 😅  https://docs.google.com/document/d/17akqrbC-w8FqEoDLvcLmrX6iw4uaVrT_xlwAl-IhEzE/edit#bookmark=kix.yb99cel3cv0k

Reply:
I see. A design bug then 😅 Every screen has a title, so I think it was missed in this case

Reply:
Sadly it's not so simple 😅 This was decided because the page only has one input, so people argued that the page title would be redundant 🤷  I can get people involved who made the decision

Reply:
Not a blocking change, We can move forward, discuss and address later. But I feel like this doesn't match with other fields.

> This was decided because the page only has one input, so people argued that the page title would be redundant

Oh, But the input label says "Date" only, so I think it might be good to have a title that explains which date it is and also align with other pages.